tests/unit/test_wagtail_hooks.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from wagtail.admin import widgets
+from wagtail.core import hooks
+
+from tests.factories.pages import ArticleFactory
+from wagtailtrans.models import TranslatablePage
+from wagtailtrans.wagtail_hooks import edit_in_language_button
+
+
+@pytest.mark.django_db
+class TestWagtailHooks:
+    def setup(self):
+        self.article = ArticleFactory()
```

Comment:
The purpose of this ``ArticlePage`` is to have a page with a relation to language which doesn't extend from ``TranslatablePage`` right? so lets call it: ``NormalPageWithLanguage`` or something like that so we have a more clear view for what purpose this page is created